docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "",
+      "poc_github": "",
+      "slug": "databricks"
+    },
+    "oss": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "",
+      "comment": ""
+    },
+    "dev_languages": {
+      "value": [
+        ""
+      ],
+      "source_url": "",
+      "comment": ""
+    },
+    "vector_launch_year": 2023,
```

Comment:
can you comment with a link to an announcement or sign-up invite from '23? I can only find https://www.databricks.com/blog/announcing-mosaic-ai-vector-search-general-availability-databricks from May 24 on the internet

Reply:
https://www.databricks.com/blog/introducing-databricks-vector-search-public-preview

Public preview link here. Since everyone who uses Vector Search is a Databricks customer, public preview is available to the same user base as GA. Usually, the difference between public preview and GA is that we don't start charging our customers for it until GA. It's similar to Google's designation of a beta release.

If you are more comfortable with the GA date, I have no problem updating the launch date to 2024, but please let me know if this explanation makes a difference to you.

Reply:
Updating to 2024 to obviate this question.